README.md

```diff
@@ -97,9 +97,12 @@ let encodedTrifleSignedDataProto = try trifleSignedData.serialize()
 ## Android SDK Usage 
 
 ```kotlin
+// App start up
+val trifleApi = TrifleApi("abc")
```

Comment:
Can you add a line as to what "abc" refers to?

Reply:
Ok